config/jobs/kubernetes-sigs/cluster-api-provider-azure/cluster-api-provider-azure-periodics.yaml

```diff
@@ -71,3 +72,39 @@ periodics:
     testgrid-dashboards: sig-cluster-lifecycle-cluster-api-provider-azure
     testgrid-tab-name: periodic-capi-e2e
     testgrid-alert-email: k8s-infra-staging-cluster-api-azure@kubernetes.io
+- name: periodic-cluster-api-provider-azure-coverage
+  interval: 12h
+  decorate: true
+  path_alias: "sigs.k8s.io/cluster-api-provider-azure"
+  extra_refs:
+    - org: kubernetes-sigs
+      repo: cluster-api-provider-azure
+      base_ref: master
+      path_alias: "sigs.k8s.io/cluster-api-provider-azure"
+    - org: kubernetes
+      repo: test-infra
+      base_ref: master
+      path_alias: k8s.io/test-infra
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/kubekins-e2e:v20200819-491a5ae-1.18
+      command:
+      - runner.sh
+      - bash
+      args:
+      - -c
+      - |
+        result=0
+        ./scripts/ci-test-coverage.sh || result=$?
+        cp coverage.* ${ARTIFACTS}
+        cd ../../k8s.io/test-infra/gopherage
+        GO111MODULE=on go build .
+        find ${ARTIFACTS} -type f -iname 'junit*.xml' -exec rm {} \;
```

Comment:
This line was just to purge all the individual junit*.xml generated because Golang coverage tool needed to be run individually for each sub package and make a lot of *.xml all in ARTF. If you are just coping coverage to ARTF I think you can exclude this line. but over all looks good to me!